meta-oe/recipes-multimedia/kmsxx/kmsxx_git.bb

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: MIT
+#
+# Copyright Leica Geosystems AG
+#
+
+SUMMARY = "C++ library for kernel mode setting"
+HOMEPAGE = "https://github.com/tomba/kmsxx"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=815ca599c9df247a0c7f619bab123dad"
+
+BRANCH = "master"
+SRC_URI = "git://github.com/tomba/kmsxx.git;protocol=https;branch=${BRANCH}"
+SRCREV = "403c756c958c2a60adc6e8fa64aa0943b4dfda4e"
+
+PACKAGES =+ "${PN}-python"
+
+DEPENDS = "libdrm libevdev fmt python3 python3-pybind11"
```

Comment:
python3-pybind11 is provided via meta-python and this layer does not depend on meta-python. Maybe this recipe should be moved into `meta-oe/dynamic-layers/meta-python/recipes-multimedia` or directly into meta-python layer